src/main/java/bio/terra/service/resourcemanagement/google/GoogleProjectService.java

```diff
@@ -104,6 +104,10 @@ public String bucketForBigQueryScratchFile(String googleProjectId) {
     return googleProjectId + "-bq-scratch-bucket";
   }
 
+  public String bucketForSnapshotExport(String googleProjectId) {
+    return googleProjectId + "-snapshot-export-bucket";
```

Comment:
What happens if/when this gets run in parallel?

Reply:
So it's not a bucket per export but per snapshot?

Reply:
yes

Reply:
got it